src/components/pages/CourseSearch.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import Requestor from '../Requestor';
+
+const CourseSearch = () => {
+  const [courses, setCourses] = useState([]);
+
+  // Get courses
+  useEffect(() => {
+    getCourses();
+    // eslint-disable-next-line
+  });
+
+  const getCourses = async () => {
+    const coursesResponse = await Requestor.get(
+      '/courses',
+      {},
+      {
```

Comment:
Let's put `DEFAULT_PARAMS` in this file instead since those params are very specific to `/courses` get request.